pegasus/sites.v3/code.org/public/hourofcode/mc.md

```diff
@@ -5,7 +5,7 @@ nav: hoc_teacher_resources_nav
 
 # MINECRAFT Hour of Code
 
-<center><iframe width="830" height="429" src="https://www.youtube.com/embed/55jADN4Y7Pg" frameborder="0" allowfullscreen></iframe></center>
+<center><iframe src="https://player.vimeo.com/video/190732678?color=d1c054&byline=0&portrait=0&badge=0" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></center>
```

Comment:
I started trying to add it for this one but it requires a youtube code and this is Minecraft's video hosted on Vimeo. Not sure if we can (or if we'd want to) upload to our own YouTube channel. Although yes, we should update the rest of these pages to use the lightbox video player eventually.